packages/aragon-cli/src/lib/acl/viewFormatter.js

```diff
@@ -0,0 +1,86 @@
+import './typedef'
+
+/**
+ * Flattens all ACL permissions from an acl object into a single array
+ * @param {AclPermissions} permissions Permissions
+ * @return {AclPermission[]} acl permissions data
+ */
+const flattenAclPermissions = permissions => {
+  const aclPermissions = []
+  for (const [to, roles] of Object.entries(permissions)) {
+    for (const [roleHash, data] of Object.entries(roles)) {
+      // Verbose assignment for data transparency
+      aclPermissions.push({
+        to,
+        role: roleHash,
+        allowedEntities: data.allowedEntities,
+        manager: data.manager,
+      })
+    }
+  }
+  return aclPermissions
+}
+
+/**
+ * Formats one acl permission data attaching human readable names to its raw data
+ * @param {AclPermission} aclPermission ACL permission data
+ * @param {App[]} apps Apps
+ * @param {KnownApps} knownApps Known apps
+ * @param {KnownRoles} knownRoles Known roles
+ * @return {AclPermissionFormatted} with human readable names if any
+ */
+const formatAclPermission = (aclPermission, apps, knownApps, knownRoles) => {
+  const {
+    to: toAddress,
+    role: roleHash,
+    allowedEntities,
+    manager: managerAddress,
+  } = aclPermission
+
+  /**
+   * Shortcut to TRY get an app name from it's proxy address
+   * If the app can't be found, returns an empty string: ''
+   * @param  {string} address Proxy address of a possible app
+   * @return {AclPermissionAppInfo} app info
+   */
+  const getAppInfo = address => {
+    const app = apps.find(({ proxyAddress }) => address === proxyAddress)
```

Comment:
I'm not sure why the kernel is printed with the address expression instead of `kernel (0x0..00)`
For what I could debug it is related with the address pass to this function be the `ZERO_ADDRESS` but couldn't understand why

Reply:
@0xGabi  could you provide steps to recreate, please?

Reply:
Yes that's fine. I was testing with a new dao creating using `dao new` and for that particular case is not displayed like this.